examples/gno.land/p/demo/grc/grc1155/util.gno

```diff
@@ -0,0 +1,18 @@
+package grc1155
+
+import (
+	"std"
+)
+
+var zeroAddress = std.Address("")
+
+func isValidAddress(addr std.Address) error {
+	if addr.String() == "" {
+		return ErrInvalidAddress
+	}
+	return nil
+}
```

Comment:
maybe change this to return a bool instead of an error? this way, the callers can do the following

```go
if !isValidAddress(from) || !isValidAddress(to) {
    return ErrInvalidAddress
}
```

in case we should distinguish the errors we can always change this as this is unexported.